sdk/iOS/test/MSCoreDataStoreTests.m

```diff
@@ -47,6 +50,54 @@ -(void)testInit
     XCTAssertNotNil(self.store, @"store creation failed");
 }
 
+// Tests MSCoreDataStore Read, Update and Delete operations are case insensitive for the Id column.
+-(void)testReadUpdateDelete_CaseInsensitiveIdOperations
+{
+    NSError *error;
+
+    // Seed data to be inserted in the table
+    NSString * const originalItemId = @"itemid";
```

Comment:
Yes, ideally all strings that are being used repeatedly should be defined as constants. I don't think it is about style. I would like to retain the const variable as it is a good coding practice.

I have changed all uses of @"TodoItem" in this file to use the const variable now. Use of cased ID constant strings is restricted to only one function as of now, so those consts are local to that function (and not defined at the class level)